v2/tools/generator/internal/astmodel/type_name.go

```diff
@@ -27,7 +27,6 @@ const (
 var armPackageDenyList = set.Make(
 	"appconfiguration",
 	"authorization",
-	"cache",
```

Comment:
Did you mean for this to be in this PR? It seems unrelated to the title of the PR

Reply:
Rebased without the extraneous changes.